core-services/prow/02_config/opendatahub-io/odh-dashboard/_prowconfig.yaml

```diff
@@ -0,0 +1,117 @@
+tide:
```

Comment:
I'm missing `merge_method` set to `squash`. See:

https://github.com/openshift/release/blob/c9e8e85464033d6ace7b05a77f9ffe43bac45754/core-services/prow/02_config/red-hat-data-services/odh-deployer/_prowconfig.yaml#L2-L3

Reply:
Thx. Updated